<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkdownBatchWidget</class>
 <widget class="QWidget" name="MarkdownBatchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,1">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Markdown Source Files</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTreeView" name="filesTreeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="importFolderPushButton">
          <property name="text">
           <string>Import Folder...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addFilesPushButton">
          <property name="text">
           <string>Add Files...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveUpPushButton">
          <property name="text">
           <string>Move Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveDownPushButton">
          <property name="text">
           <string>Move Down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removePushButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeAllPushButton">
          <property name="text">
           <string>Remove All</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Export Folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="exportFolderLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browserExportFolderPushButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="useCssCheckBox">
          <property name="text">
           <string>Use CSS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cssComboBox"/>
        </item>
        <item>
         <widget class="QCheckBox" name="mergeHrCheckBox">
          <property name="text">
           <string>Insert horizontal line between merged files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mergePageBreakCheckBox">
          <property name="text">
           <string>Insert page break between merged files</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>37</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="splitHtmlPushButton">
        <property name="text">
         <string>Export Separated HTML</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="splitPdfPushButton">
        <property name="text">
         <string>Export Separated PDF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mergetHtmlPushButton">
        <property name="text">
         <string>Export Merged HTML...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mergePdfPushButton">
        <property name="text">
         <string>Export Merged PDF...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mergePrintPreviwPushButton">
        <property name="text">
         <string>Print Preview Merged...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mergetPrintPushButton">
        <property name="text">
         <string>Print Merged</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Log</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="logPlainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
 </widget>
 <tabstops>
  <tabstop>filesTreeView</tabstop>
  <tabstop>importFolderPushButton</tabstop>
  <tabstop>addFilesPushButton</tabstop>
  <tabstop>removeAllPushButton</tabstop>
  <tabstop>useCssCheckBox</tabstop>
  <tabstop>cssComboBox</tabstop>
  <tabstop>exportFolderLineEdit</tabstop>
  <tabstop>browserExportFolderPushButton</tabstop>
  <tabstop>splitHtmlPushButton</tabstop>
  <tabstop>splitPdfPushButton</tabstop>
  <tabstop>mergetHtmlPushButton</tabstop>
  <tabstop>mergePdfPushButton</tabstop>
  <tabstop>mergePrintPreviwPushButton</tabstop>
  <tabstop>mergetPrintPushButton</tabstop>
  <tabstop>logPlainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
